### ISSUE TITLE
font.layout can't parse family emoji

### ISSUE THREAD

**Issue by samjohn** [devongovett/fontkit#20]
```
var family = font.layout('👨‍👩‍👧‍👦👨‍👩‍👧👨‍👩‍👦‍👦');
```

Produces:
```
TypeError: Cannot read property 'codePoints' of undefined
    at AATMorxProcessor.processLigature (/Users/samanthajohn/Development/hopscotch-community/submodules/webplayer/node_modules/fontkit/src/aat/AATMorxProcessor.js:254:75)
    at /Users/samanthajohn/Development/hopscotch-community/submodules/webplayer/node_modules/fontkit/src/aat/AATMorxProcessor.js:4:59
    at AATStateMachine.process (/Users/samanthajohn/Development/hopscotch-community/submodules/webplayer/node_modules/fontkit/src/aat/AATStateMachine.js:55:11)
    at AATMorxProcessor.processSubtable (/Users/samanthajohn/Development/hopscotch-community/submodules/webplayer/node_modules/fontkit/src/aat/AATMorxProcessor.js:124:27)
    at AATMorxProcessor.process (/Users/samanthajohn/Development/hopscotch-community/submodules/webplayer/node_modules/fontkit/src/aat/AATMorxProcessor.js:81:18)
    at LayoutEngine.substitute (/Users/samanthajohn/Development/hopscotch-community/submodules/webplayer/node_modules/fontkit/src/layout/LayoutEngine.js:99:28)
    at LayoutEngine.layout (/Users/samanthajohn/Development/hopscotch-community/submodules/webplayer/node_modules/fontkit/src/layout/LayoutEngine.js:84:21)
    at TTFFont.layout (/Users/samanthajohn/Development/hopscotch-community/submodules/webplayer/node_modules/fontkit/src/TTFFont.js:218:33)
    at repl:1:19
    at REPLServer.defaultEval (repl.js:132:27)
```

With the `Apple Color Emoji.ttf` font available on Macs.

Note: chrome also cannot really parse the family emoji. To see this properly look on safari.

**Comment by samjohn** [comment]
The best behavior here would be to find the correct ligature. But acceptable behavior would be to generate non-ligature glyphs, as Chrome does.

**Comment by devongovett** [comment]
Should be fixed by 8961a649bbb15f024b43d2c83c0406e3db086d9d.
